medical_practitioner/models/medical_practitioner.py

```diff
@@ -22,3 +28,30 @@ class MedicalPractitioner(models.Model):
                    ('external', 'External Entity')],
         readonly=False,
     )
+    code = fields.Char(
+        string='Internal ID',
+        help='Unique ID for this physician',
+        required=True,
+        default=lambda s: s.env['ir.sequence'].next_by_code(s._name + '.code'),
+    )
+    specialty_ids = fields.Many2many(
+        string='Specialties',
+        comodel_name='medical.specialty',
+    )
+
+    @api.model
+    def _get_default_image_path(self, vals):
+        res = super(MedicalPractitioner, self)._get_default_image_path(vals)
+        if res:
+            return res
+
+        practitioner_gender = vals.get('gender', 'male')
```

Comment:
To me, it seems like the image will always be for 'male' gender. Is that what you wanted?
IMO nothing specifies gender in Practitioner and titles aren't considered.

Reply:
Practitioners have a ``gender`` field that's inherited from ``res.partner`` through ``medical.abstract.entity``.

Reply:
But is there the option to define the gender when creating a practitioner? The rest seems great to me!

Reply:
Ah. Good catch. I've added the field to the `medical.abstract.entity` form view so that it can be set during practitioner creation.